scriptmodules/ports/ikemen-go.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="ikemen-go"
+rp_module_desc="I.K.E.M.E.N GO - Clone of M.U.G.E.N to the Go programming language"
+rp_module_licence="MIT https://raw.githubusercontent.com/Windblade-GR01/Ikemen-GO/master/License.txt"
+rp_module_help="Copy characters, stages, screenpacks, etc. to $romdir/ports/ikemen-go\n\nConfig files can be found at $configdir/ports/ikemen-go/save"
+rp_module_section="exp"
+
+function depends_ikemen-go() {
+    getDepends golang-go libgl1-mesa-dev xinit xorg xorg-dev libopenal1 libopenal-dev libgtk-3-dev libasound2-dev freeglut3-dev
```

Comment:
The dependencies should only include the development packages. As they depend on the binary libraries they can be excluded. Eg `libopenal1`. Also xorg package depends on xinit so that's not needed.
Is xorg-dev needed? Maybe can tweak this a bit to minimise dependencies

Reply:
`xorg-dev` is [apparently needed for GLFW](https://www.glfw.org/docs/latest/compile.html#compile_deps) on Debian distributions, but the other non-`dev` packages could probably be removed without issue.

Reply:
Okay, I just tested: removing `xorg` is okay, but removing `xorg-dev` and replacing it with `libglfw3-dev` causes IKEMEN GO to not be able to start, with the following runcommand.log:
```
Parameters: 
Executing: xinit /dev/shm/retropie_xinitrc -- vt1 -keeptty

xinit: unable to run server "X": No such file or directory
Use the -- option, or make sure that /usr/bin is in your path and
that "X" is a program or a link to the right type of server
for your display.  Possible server names include:

    Xorg        Common X server for most displays
    Xvfb        Virtual frame buffer
    Xfake       kdrive-based virtual frame buffer
    Xnest       X server nested in a window on another X server
    Xephyr      kdrive-based nested X server
    Xvnc        X server accessed over VNC's RFB protocol
    Xdmx        Distributed Multi-head X server

xinit: giving up
xinit: unable to connect to X server: Connection refused
xinit: server error
```
So it does appear that  `xorg-dev` is necessary to get xinit to start.
On a related note, I tested and found that `freeglut3-dev` is seemingly unnecessary.

Reply:
Okay, further testing seems to indicate that `xorg` will suffice and that `xorg-dev` is overkill.

Reply:
`xterm` would also be needed to be installed - it's not included in `xorg` and it's used by the launching script.

I've compiled the new module ran it briefly from ES (Pi4) and the video it's noticeably slow (like it's using software rendering). The lack of fullscreen may be remedied through configuration, I haven't fiddled with it.
Maybe I need to re-build again, but I doubt is the dependencies that may be causing this. Any method to check whether the engine uses HW acceleration ? The log file from the install folder is empty.

Reply:
I believe that IKEMEN GO doesn't have any form of software rendering; i.e. it only has an OpenGL-based renderer; one not optimized specifically for the Pi, which is likely why it's slow.

Fullscreen can indeed be enabled in-game via the video settings (or alternatively editing `save/config.json` and manually setting fullscreen to true); the only reason I haven't set it to on by default (aside from it crashing composite-out mode) is that IKEMEN doesn't include a default config.json, generating it on first run instead.

From what I can tell, the log file only gets written to in the event of a crash.  I don't think IKEMEN GO has any form of `--verbose` option either, though I could be wrong on that.